Doc/library/mmap.rst

```diff
@@ -81,7 +81,11 @@ To map anonymous memory, -1 should be passed as the fileno along with the length
    private copy-on-write mapping, so changes to the contents of the mmap
    object will be private to this process, and :const:`MAP_SHARED` creates a
    mapping that's shared with all other processes mapping the same areas of
-   the file.  The default value is :const:`MAP_SHARED`.
+   the file. :const:`MAP_ALIGNED_SUPER` (FreeBSD only) allows to map to
+   pages of 2MB (x86) or so, it is enabled by default, at boot time,
```

Comment:
```suggestion
   pages of 2 MiB (x86) or so, it is enabled by default, at boot time,
```

Is it x86 (32-bit) or x86-64 (64-bit)?

Reply:
both.

Reply:
Write maybe (x86, x86-64) in this case... But I'm not sure if Python documentation should go too far in the description, since FreeBSD may decide to add support for MAP_ALIGNED_SUPER for other platforms.

Reply:
It supports more than x86 already in fact but you re right I ll try to find more concise wording